examples/data-sources/hpegl_metal_available_resources/README.md

```diff
@@ -139,12 +139,18 @@ storage-pools = [
     "Name": "abtest",
     "LocationID": "c8b3c5a7-f81d-453a-af3a-1e6d78291bd5",
     "Description": ""
+    "StoragePoolIDs": [
+        "bdbbb385-2ee3-4e71-a60a-ba712cdda37e"
+      ]
   },
   {
     "ID": "dd253c2a-defb-41c7-b23d-f9a937c37da0",
     "Name": "abose-tf-test",
     "LocationID": "c8b3c5a7-f81d-453a-af3a-1e6d78291bd5",
     "Description": ""
+        "StoragePoolIDs": [
+        "bdbbb385-2ee3-4e71-a60a-ba712cdda37e"
```

Comment:
Spacing is not lined up correctly. Also, can a StoragePool have multiple VolumeCollections?

Reply:
Yes. Many-to-many in both directions. There is no _direct_ relationship between a pool and collection. Storage pool is defined for a number of storage systems. VC is defined for a storage group. VC can be used for volumes in any of those  pools. And from any of the storage pool you can know which VCs are applicable in that storage group.

Reply:
Space alignment is not right. StoragePoolIDs should line up with Description. The first example is correct, but this second example is still not right.